pdc_client/runner.py

```diff
@@ -110,14 +110,19 @@ def load_plugins(self):
             for name in os.listdir(dir):
                 if not name.endswith('.py') or name not in plugins_set:
                     continue
-                file, pathname, description = imp.find_module(name[:-3], [dir])
-                plugin = imp.load_module(name[:-3], file, pathname, description)
-                self.logger.debug('Loaded plugin {0}'.format(name[:-3]))
-                self.raw_plugins.append(plugin)
-                if hasattr(plugin, 'PLUGIN_CLASSES'):
-                    for p in plugin.PLUGIN_CLASSES:
-                        self.logger.debug('Instantiating {0}'.format(p.__name__))
-                        self.plugins.append(p(self))
+                try:
+                    file, pathname, description = imp.find_module(name[:-3], [dir])
+                    plugin = imp.load_module(name[:-3], file, pathname, description)
+                    self.logger.debug('Loaded plugin {0}'.format(name[:-3]))
+                    self.raw_plugins.append(plugin)
+                    if hasattr(plugin, 'PLUGIN_CLASSES'):
+                        for p in plugin.PLUGIN_CLASSES:
+                            self.logger.debug('Instantiating {0}'.format(p.__name__))
+                            self.plugins.append(p(self))
+                except:
+                    print('Failed to load plugin {0}'.format(name[:-3]))
```

Comment:
Omit `print()` for status and error messages. And try to show more information.
```python
except Exception as e:
    self.logger.error('Failed to load plugin "{0}": {1}'.format(name[:-3], e))
```

Reply:
I think use print here more makes sense, we have no need to show a "Error:pdc: ......" when ct meets error in command lines, but the error info should be included.
Changed.

Reply:
With `print()` you're printing on stdout by default, I wouldn't expect any errors there. Let logger handle the details. Also it makes messages to follow some format and I can grep errors easily.

Reply:
# Also it makes messages to follow some format and I can grep errors easily.
Make sense, change

Reply:
Changed, please help to review, will commit another patch to change other print